src/python/qeqiskit/simulator/simulator.py

```diff
@@ -128,6 +128,8 @@ def run_circuit_and_measure(self, circuit, **kwargs):
         for bitstring in raw_counts.keys():
             reversed_counts[bitstring[::-1]] = raw_counts[bitstring]
 
+        self.number_of_circuits_run += 1
+        self.number_of_jobs_run += 1
```

Comment:
Similar to before, I think `super().run_circuit_and_meeasure(circuit)` would be best

Reply:
Addressed in [d81040](https://github.com/zapatacomputing/qe-qiskit/pull/18/commits/d8104090971fb60f46de371cafe4e6e745c211d3).